packages/tour-kit/src/utils/index.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Helper to convert CSV of `classes` to an array.
+ *
+ * @param classes String or array of classes to format.
+ * @returns Array of classes
+ */
+export function classParser( classes?: string | string[] ): string[] | null {
+	if ( classes?.length ) {
+		return classes?.toString().split( ',' );
```

Comment:
I think we can skip the optional chaining here, as we already verified that `classed` contains something 👍

Reply:
I added that to make sure the function returns `null` in the event that *classes* is either `null` or an empty array.

Without `?.length` the function returns `['']` if given an empty array
Without making it optional the function errors when given a `null` value

Thoughts?

Reply:
Oh I was referring to `return classes?.toString().split( ',' );`, the optional chaining there!

Reply:
Ahhhh sorry haha misread that